app/services/invite_schools.rb

```diff
@@ -61,6 +61,42 @@ def send_ministerial_letters
     end
   end
 
+  def invite_to_beta(school_urns)
+    start_url = Rails.application.routes.url_helpers.root_url(
+      host: Rails.application.config.domain,
+      **UTMService.email(:june_private_beta, :private_beta),
+    )
+
+    school_urns.each do |urn|
+      school = School.find_by(urn: urn)
+      if school.nil?
+        logger.info "School not found, urn: #{urn} ... skipping"
+        next
+      end
+
+      if FeatureFlag.active?(:induction_tutor_manage_participants, for: school)
```

Comment:
And how do we turn it on, and for what schools?

Reply:
Line 88